site/src/comparison.rs

```diff
@@ -257,18 +259,30 @@ pub async fn compare_given_commits(
     }))
 }
 
-/// Data associated with a specific date
+/// Data associated with a specific artifact
 #[derive(Debug, Clone, Serialize)]
 pub struct DateData {
+    /// The artifact in question
+    pub commit: String,
```

Comment:
This should probably be `artifact: ArtifactId` since `commit` is wrong (it might be an artifact "tag" and not a commit sha) and there's no need for it to be a string at this point.